lib/clearance/session.rb

```diff
@@ -2,7 +2,7 @@
 
 module Clearance
   class Session
-    REMEMBER_TOKEN_COOKIE = 'remember_token'.freeze
+    REMEMBER_TOKEN_COOKIE = Clearance.configuration.cookie_name.freeze
```

Comment:
Can we remove this constant entirely, or is it considered part of the public API?